mathics/builtin/graphics.py

```diff
@@ -694,11 +777,45 @@ def apply(self, c1, c2, evaluation, options):
             if not compute:
                 evaluation.message('ColorDistance', 'invdist', distance_function)
                 return
+        elif distance_function.has_form('List', 2):
+            if (isinstance(distance_function.leaves[0], String)
+            and distance_function.leaves[0].get_string_value() == 'CMC'):
+                if isinstance(distance_function.leaves[1], String):
+                    if distance_function.leaves[1].get_string_value() == 'Acceptability':
```

Comment:
It's safe not to check that everything is a string here because `get_string_value` (just like `get_int_value`) will return `None` if it's not a `String` instance.